spec/controllers/webhooks_controller_spec.rb

```diff
@@ -124,6 +124,23 @@
 
     end
 
+    context 'with a change update event' do
+
+      let(:optin_json) { File.read("spec/fixtures/change_notification_request.json") }
+
+      context 'with valid params' do
+        before { post :callback, optin_json }
+        it { expect(response).to have_http_status(200) }
```

Comment:
Yeah, I stuck to this because it seemed to be the convention used in the specs already ><